packages/experimental/Menu/src/Menu/useMenu.ts

```diff
@@ -9,8 +9,10 @@ export const useMenu = (props: MenuProps): MenuState => {
   const isSubmenu = context.triggerRef !== null;
   const isControlled = typeof props.open !== 'undefined';
   const [open, setOpen] = useMenuOpenState(isControlled, props);
+  const openOnHover = isSubmenu;
```

Comment:
We don't want to do something like `const openOnHover = props.openOnHover || isSubmenu` ? or `(props.openOnHover !=  undefined) ? props.openOnHover : isSubmenu`

Reply:
The `...props` below will provide openOnHover behavior if someone wants it for a parent menu or override a submenu's default openOnHover if someone wants to disable that behavior. But this is probably unclear, I can add a comment here on intended behavior.

Reply:
Yeah I realized that after finishing the review. I think if we want to keep the "use the spread to override the default" we should add a comment. I'm biased because I don't think it's obvious to many that spreads can overwrite previous props. Feel free to disagree and keep it as is if you prefer the brevity.